docs/atomic-cluster.1.md

```diff
@@ -0,0 +1,65 @@
+% ATOMIC(1) Atomic Man Pages
+% Matthew Barnes
+% April 2016
+# NAME
+atomic-cluster - Manage cluster hosts
+
+# SYNOPSIS
+**atomic cluster [OPTIONS] SUBCOMMAND**
+
+**atomic cluster** will issue commands to a Commissaire server over HTTP.
+
+Commissaire allows administrators of a Kubernetes, Atomic Enterprise or
+OpenShift installation to perform administrative tasks without the need
+to write custom scripts or manually intervene on systems.
+
+# NOTE
+The `cluster` subcommand is only available when the Commissaire client
+module `commctl` is installed.
+
+# OPTIONS
+**-h** **-help**
+  Print usage statement.
+
+# SUBCOMMANDS
+**get cluster** CLUSTER_NAME
+  Report status of the specified cluster.
```

Comment:
Hmm... I agree that particular case reads better.  But there's some other types of things you can "get" (or "create" or "delete") besides cluster objects and I'm worried it would make parsing tricky -- or even ambiguous if we add more types of things to "get" in the future.

An easier solution is to rename the `atomic cluster` part, though the best I can think of is `atomic multihost`.

@ashcrow: Any thoughts here?